docs/installation/install-zoom-plugin.md

```diff
@@ -4,19 +4,20 @@ description: Get the Mattermost plugin installed as a first step
 
 # Install Zoom Plugin
 
-## Install via Plugin Marketplace \(Recommended\)
+## Install via Marketplace \(Recommended\)
 
-1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost.
-2. Search for "Zoom" or manually find the plugin from the list and click **Install**.
+1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
+2. Search for "Zoom" or manually find the plugin from the list and select **Install**.
 3. After the plugin has downloaded and been installed, follow the instructions on [Zoom Configuration](zoom-configuration/).
 4. After finishing the [Zoom Configuration](zoom-configuration/), follow the instructions on [Mattermost Setup](mattermost-setup.md).
 
-## \(Alternative\) Install via Manual Upload
+## \(Alternative\) Install via manual upload
 
-If your server doesn't have internet access, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console &gt; Plugin Management**. The binary releases on the page above, are the same as used by the Plugin Marketplace.
-
-1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost.
-2. Search for "Zoom" or manually find the plugin from the list and click **Install**.
-3. After the plugin has downloaded and been installed, follow the instructions on [Zoom Configuration](zoom-configuration/).
-4. After finishing the [Zoom Configuration](zoom-configuration/), follow the instructions on [Mattermost Setup](mattermost-setup.md).
+If your server doesn't have internet access, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console > Plugin Management**. The binary releases on the page above, are the same as used by the Marketplace.
 
+1. Go to **Main Menu > Marketplace** in Mattermost.
```

Comment:
Is this correct? Shouldn't this be something like "Go to the system console, manage plugins"?